dist/src/PinCodeEnter.js

```diff
@@ -17,15 +17,16 @@ class PinCodeEnter extends React.PureComponent {
                 this.props.endProcessFunction(pinCode);
             }
             else {
+                let pinValidOverride = undefined;
                 if (this.props.handleResult) {
-                    this.props.handleResult(pinCode);
+                    pinValidOverride = Promise.resolve(this.props.handleResult(pinCode));
```

Comment:
I've tried this and it doesn't resolve the value correctly, pinValidOverride ends up as `Promise {_40: 0, _65: 0, _55: null, _72: null}`
Since the endProcess is an async function, you can use await which will correctly resolve to the boolean value.
```
if (this.props.handleResult) {
    pinValidOverride = await this.props.handleResult(pinCode);
}```

Reply:
yes, thank you, but the fix is already applied. you got old version somehow.

Reply:
Hello leo, is this repository still maintained?

Is this branch going to get merged?

I need external pin validation but any attempts to modify the behaviour, e.g. using        
`handleResultEnterPin={this.onPinScreenCompletion}`

just results in a Dead pin-screen, no errors, text up top disappears, buttons become non-responsive.
The only callback that seems to work is the `storePinFunction` override but that's useless if I can't set it to store and compare to hashed pins too...

Reply:
Hi AJO, i am using my fix since i need it by using my fork instead of main repo.
I do not maintain this repo and couldnt tell you if it will ever be merged.